scripts/integration-test

```diff
@@ -14,4 +14,4 @@ if [ ! -e ../dist/artifacts/initrd ]; then
     ../scripts/dev
 fi
 
-go test -timeout 30m
+go test -v -check.vv -timeout 4h
```

Comment:
Adding `-check.vv` should be in a different PR so we could discuss the merit of that independently.